cpg-language-java/src/test/kotlin/de/fraunhofer/aisec/cpg/frontends/java/JavaLanguageFrontendTest.kt

```diff
@@ -807,52 +807,44 @@ internal class JavaLanguageFrontendTest : BaseTest() {
             TestUtils.analyze(listOf(file), file.parentFile.toPath(), true) {
                 it.registerLanguage(language)
             }
-        val tu =
-            findByUniqueName(result.components.flatMap { it.translationUnits }, file.toString())
-
-        val record = tu.getDeclarationAs(0, RecordDeclaration::class.java)
+        val record = result.records["Operators"]
         assertNotNull(record)
+        assertFalse { record.methods.isEmpty() }
+
+        val mainMethod = record.methods["main"]
 
-        val mainMethod = record.methods[0]
+        val expressionLists = mainMethod.mcalls
+        assertEquals(6, expressionLists.size)
 
-        val intOperationsList = mainMethod.getBodyStatementAs(0, MemberCallExpression::class.java)
-        assertNotNull(intOperationsList)
+        val intOperationsList = expressionLists[0]
         assertEquals(14, intOperationsList.arguments.size)
         assertTrue { intOperationsList.arguments.all { language.builtInTypes["int"]!! == it.type } }
```

Comment:
small nitpick: We try to avoid the forced non-null `!!` in Kotlin. The "canonical" way to do would be to use `primitiveType("int")` in order to retrieve the type. But that needs a language context, so you would need to do something like this:

```Kotlin
val main = record.methods["main"]
with(main) {
   val expressions = it.mcalls
   val intOperations = expressions[0]
   assertTrue { intOperations.arguments.all { primitiveType("int") == it.type } }
}
```

You can also use `with(language)`

Reply:
I see – done.